### PR TITLE
chore[models]: GrantModel

### PR DESCRIPTION
**Pull request by armin3731**
I updated models so that GrantModel can be used inside apps

### CODE REVIEW

**Comment by mmRoshani** [comment]
There is another PR in progress, please wait till the [PR#8](https://github.com/RahkarSanat/kiz-sdk-js/pull/8) is complete.
That makes us sure there are no merge conflicts.

Best regards.

File: src/common/models/grant.model.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  subject!: string;
+  action!: Action;
+  object!: string;
+  field?: string[];
+  filter?: string[];
+  location?: string[];
+  time?: GrantTimeModel[];
+}
+
+export class GrantTimeModel {
```
Comment (mmRoshani) [inline src/common/models/grant.model.ts]: Due to the weakness of `GrantTimeModel`, do you need it outside the SDK?
if not remove the `export` keyword.

**Review by mmRoshani — COMMENTED** [review]
there is just one question

**Comment by armin3731** [comment]
It wasn't necessary, therefore I removed it. Thanks for the tip 👍 🍻

**Pull request merged**